lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -169,10 +165,10 @@ def add_refund_customer_data(post, options)
           post['customer']['address'] = {
             'countryCode' => address[:country]
           }
-          post['customer']['contactDetails'] = {
-            'emailAddress' => options[:email],
-            'phoneNumber' => address[:phone]
-          }
+          post['customer']['contactDetails']['emailAddress'] = options[:email] if options[:email]
```

Comment:
Same here; wondering if we need `post['customer']['contactDetails'] ||= {}` to be safe.